string[] meshNames
geometry_msgs/Pose[] poses
---
int32[] errcodes
string[] errmsgs
string[] instanceNames
